packages/volto/src/components/manage/Blocks/Block/Order/Item.jsx

```diff
@@ -91,6 +99,9 @@ export const Item = forwardRef(
           <button
             ref={ref}
             {...handleProps}
+            style={{
```

Comment:
instead of having a visibility style it would make more sense to me to have the "disabled" property added and this way you also get a screen reader compatible information on why you can't interact with the button

Reply:
Looks good, but there are other places were we have used visibilty style will update it today

Reply:
@ichim-david I did added disabled property but removing visiblity style it's didn't seemed to respect the required functionality, If you would like to add more changes let me know